google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/AddGeoTarget.java

```diff
@@ -103,7 +103,11 @@ public static void main(String[] args) {
    * @param geoTargetConstantId the geo target constant ID to add to the extension feed item.
    */
   private void runExample(
-      GoogleAdsClient googleAdsClient, long customerId, long feedItemId, long geoTargetConstantId) {
+      GoogleAdsClient googleAdsClient, long customerId, long feedItemId, Long geoTargetConstantId) {
+    // Uses a default geoTargetConstantId value if none is provided.
+    geoTargetConstantId =
+        (geoTargetConstantId == null) ? GEO_TARGET_CONSTANT_ID : geoTargetConstantId;
```

Comment:
Could just set this at the field declaration, and allow the flags parser to override.

Reply:
I may not be understanding you here, but I think that was the previous behavior that did not work.

Reply:
+1

Reply:
I meant setting the default on line 53.

Reply:
Got it. Fixed.